src/runtime_src/tools/xclbinutil/unittests/MCS/MCSSection.py

```diff
@@ -0,0 +1,168 @@
+import subprocess
+import os
+import argparse
+from argparse import RawDescriptionHelpFormatter
+import filecmp
+import json
+import binascii
```

Comment:
Sort

Reply:
Updated all of the python unit test scripts to have their imports sorted.